src/components/Letters.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import Letter from './Letter'
+
+class Letters extends Component {
+    render() {
+        return (
+            <div>
+                <div>Available letters</div>
+                <div>{Object.keys(this.props.letterStatus).map(k => <Letter letter={k} className={this.props.letterStatus[k] === true? "striked" : k} 
```

Comment:
brining it to the next line down like I showed before would make this look cleaner